README.md

```diff
@@ -133,12 +133,7 @@ cloning the git repo), execute:
 pip install .
 ```
 
-or for installing in [development mode](https://pip.pypa.io/en/latest/cli/pip_install/#install-editable):
-
-
-```sh
-python -m pip install -ve . --no-build-isolation --config-settings=editable-verbose=true
-```
+or for installing in [development mode](https://pip.pypa.io/en/latest/cli/pip_install/#install-editable).
```

Comment:
Can you delete this block?

The line below telling users how to install from source should be enough.